jest-mongodb-setup.js

```diff
@@ -0,0 +1,27 @@
+import { MongoClient } from 'mongodb';
+
+let connection
+beforeAll(async () => {
+  process.env.MONGODB_URI = global.__MONGO_URI__
+
+  connection = await MongoClient.connect(process.env.MONGODB_URI, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true
+  });
```

Comment:
Does this connect to our local db's..?

Reply:
Not the DB we use for local development. It's a separate test database that's cleaned up between each test and doesn't save any data.